### PR TITLE
cmd: add ca-certificates to Docker images

### PR DESCRIPTION
**Pull request by stefanbenten**
What/Why: This change removes the manually override of the ca-certificates file
in the storagenode docker image, which causes issues and is error prone.
It now installs ca-certificates in all containers to be future proofed.

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: cmd/uplink/Dockerfile [hunk 1]
```diff
@@ -9,6 +9,7 @@ ENV CONF_PATH=/root/.local/storj/uplink \
     SATELLITE_ADDR=
```
Comment (stefanbenten) [inline cmd/uplink/Dockerfile:9]: @egonelbre We should check, if those env variables are still used/valid.

**Review by egonelbre — APPROVED** [review]

File: cmd/uplink/Dockerfile (hunk 1 above)
Comment (egonelbre) [reply cmd/uplink/Dockerfile:9]: no clue

File: cmd/storagenode/Dockerfile [hunk 2]
```diff
@@ -5,7 +5,7 @@ ARG GOARCH
 ENV GOARCH ${GOARCH}
 EXPOSE 28967
 WORKDIR /app
-COPY resources/certs.pem /etc/ssl/certs/ca-certificates.crt
```
Comment (ifraixedes) [inline cmd/storagenode/Dockerfile:8]: I'm wondering what this certificate in our repository was about. It was a copy of the `ca-certificates.crt`?

**Review by ifraixedes — APPROVED** [review]

File: cmd/storagenode/Dockerfile (hunk 2 above)
Comment (stefanbenten) [reply cmd/storagenode/Dockerfile:8]: As answered in standup, this was a single root CA of lets encrypt that was needed in the past. We entirely overwrote the default CA list.

**Pull request merged**